src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -2091,18 +2091,11 @@ public enum AspNetHostingPermissionLevel
         Unrestricted = 600,
     }
 }
+
+#if netcoreapp
```

Comment:
Can you instead put this in a separate file System.Security.Permissions.netcoreapp.cs and conditionally include it?

Reply:
Thanks for doing this.  I see now plenty of other ifdefs in this file that could be cleaned up, but I appreciate not making this worse.